### ISSUE TITLE
chokidar@^4 causes pnpm trustPolicy no-downgrade failure (ERR_PNPM_TRUST_DOWNGRADE)

### ISSUE THREAD

**Issue by arantespp** [TypeStrong/fork-ts-checker-webpack-plugin#895]
## Summary
Projects using strict pnpm supply-chain policies (`trustPolicy: no-downgrade`) fail install because this package depends on `chokidar@^4.0.1`, which currently resolves to `chokidar@4.0.3` and is flagged as a trust downgrade.

## Error

```text
[ERR_PNPM_TRUST_DOWNGRADE] 1 lockfile entries failed verification:
  chokidar@4.0.3 High-risk trust downgrade for "chokidar@4.0.3" (possible package takeover)
```

## Dependency path

`@storybook/builder-webpack5 -> fork-ts-checker-webpack-plugin@9.1.0 -> chokidar@^4.0.1`

## Why this matters
With pnpm v11 security settings (`minimumReleaseAge`, `blockExoticSubdeps`, `trustPolicy: no-downgrade`), this blocks `pnpm install` unless users explicitly relax policy (`trustPolicyExclude`).

## Request
Could you provide a path that avoids this trust-downgrade block for strict-policy users? For example:

- update dependency strategy away from the affected `chokidar` line, or
- document recommended compatibility/security guidance for package managers enforcing trust-level checks.

Happy to test any candidate version quickly.
